### ISSUE TITLE
refactor validation/test split logic

### ISSUE THREAD

**Issue by frolovsa** [frolovsa/model_error_correction#16]
right now the validation set is the tail end of the training data. suggested. do a random shuffle instead
